paasta_tools/chronos_serviceinit.py

```diff
@@ -314,18 +314,33 @@ def status_chronos_jobs(client, jobs, job_config, verbose):
         provided by chronos_tools.load_chronos_job_config().
     :param verbose: int verbosity level
     """
-    if jobs == []:
+    # Verbose mode shows previous versions.
+    matching_jobs = chronos_tools.lookup_chronos_jobs(
+        service=service,
+        instance=instance,
+        client=client,
+        include_disabled=True,
+    )
+    sorted_matching_jobs = chronos_tools.sort_jobs(matching_jobs)
+    job_config = chronos_tools.load_chronos_job_config(
+        service=service,
+        instance=instance,
+        cluster=cluster,
+        soa_dir=soa_dir,
+    )
+    if sorted_matching_jobs == []:
         return "%s: chronos job is not set up yet" % PaastaColors.yellow("Warning")
     else:
         output = []
         desired_state = job_config.get_desired_state_human()
-        output.append("Desired:    %s" % desired_state)
-        for job in jobs:
+        output.append("    Desired:    %s" % desired_state)
```

Comment:
Instead of adding the spaces in front everywhere in chronos_serviceinit, you could copy https://github.com/Yelp/paasta/blob/f5f9f0f3d214b90ca0384a9d9ffe349531dd749d/paasta_tools/cli/cmds/status.py#L621-L622 into print_chronos_status.